src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -127,15 +121,13 @@ public void setHeadless(boolean headless) {
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes